doc/classes/CanvasGroup.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CanvasGroup" inherits="Node2D" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		Merges several 2D nodes into a single draw operation.
 	</brief_description>
 	<description>
+		Child [CanvasItem] nodes of a [CanvasGroup] are drawn as a single object. It allows to e.g. draw overlapping translucent 2D nodes without blending (set [member CanvasItem.self_modulate] property of [CanvasGroup] to achieve this effect). It also affects clipping.
```

Comment:
I need some help to clarify the clipping. I see it has an effect, but hard to say what it does exactly.

Reply:
@clayjohn

Reply:
I'm not sure what you mean about the clipping. CanvasGroup draws all its children as one CanvasItem, but it doesn't really do any extra clipping or anything

Reply:
idk seems different, is this a bug?
![godot windows editor dev x86_64_yXf2ekUi9E](https://user-images.githubusercontent.com/2223172/203134333-d3444b06-5d3f-48c1-b87a-5c61ad5c843e.gif)
I think the original PR mentioned clipping too, but maybe it wasn't related to CanvasGroup.

Reply:
Does "icon" have clip_children enabled?

Reply:
> Does "icon" have clip_children enabled?

Yes.

Reply:
Okay, that makes sense. I would replace the last sentence as follows:

```suggestion
		Child [CanvasItem] nodes of a [CanvasGroup] are drawn as a single object. It allows to e.g. draw overlapping translucent 2D nodes without blending (set [member CanvasItem.self_modulate] property of [CanvasGroup] to achieve this effect). Since [CanvasGroup] and [member CanvasItem.clip_children] both utilize the backbuffer, children of a [CanvasGroup] who have their [member CanvasItem.clip_children] set to anything other than [constant CanvasItem.ClipChildrenMode.CLIP_CHILDREN_DISABLED] will not function correctly.
```